### PR TITLE
Solomon alma changes

### PR DESCRIPTION
**Pull request by SolomonAlma**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2170)

### CODE REVIEW

**Comment by SolomonAlma** [comment]
Please review this, @casey-collab

File: hebrew/esti/esti.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+נחוש להיות הישראלי הראשון שעשה את שתי דוגמניות העל וזכה לספר על זה, אתה מרים טלפון לאסתי גינזבורג
```
Comment (casey-collab) [inline hebrew/esti/esti.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: hebrew/esti/maserati.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+אתה מחליט שעם הכוח החדש שלך, יש לך אחריות לנוע ברחובות תל אביב ברכב שראוי לך.
```
Comment (casey-collab) [inline hebrew/esti/maserati.md]: This file has also been added to a directory that already contained a page.

File: hebrew/esti/esti.md (hunk 1 above)
Comment (casey-collab) [inline hebrew/esti/esti.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: hebrew/esti/maserati.md (hunk 2 above)
Comment (casey-collab) [inline hebrew/esti/maserati.md]: This page is also unreachable.

File: hebrew/maserati/maserati.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+נחוש להיות הישראלי הראשון שעשה את שתי דוגמניות העל וזכה לספר על זה, אתה מרים טלפון לאסתי גינזבורג
```
Comment (casey-collab) [inline hebrew/maserati/maserati.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by SolomonAlma** [comment]
Please review this, @casey-collab

File: hebrew/bar/bar.md [hunk 4]
```diff
@@ -7,4 +7,8 @@
 
 עישנת לעצמך איזה ג'וינט טוב.
 
-מה עכשיו?
\ No newline at end of file
+מה עכשיו?
+
+[תתקשר לאסתי גינזבורג](esti/esti.md)
```
Comment (casey-collab) [inline hebrew/bar/bar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: hebrew/bar/bar.md [hunk 5]
```diff
@@ -7,4 +7,8 @@
 
 עישנת לעצמך איזה ג'וינט טוב.
 
-מה עכשיו?
\ No newline at end of file
+מה עכשיו?
+
+[תתקשר לאסתי גינזבורג](esti/esti.md) 
+
+[תתקשר לסוכנות מזראטי](maserati/maserati.md)
```
Comment (casey-collab) [inline hebrew/bar/bar.md]: This link is also invalid.

File: hebrew/esti/esti.md (hunk 1 above)
Comment (casey-collab) [inline hebrew/esti/esti.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**